build/litsearch/getInternalData.py

```diff
@@ -175,14 +175,51 @@ def dateStringFromDateParts(row):
     return citationsDF
 
 
+# Pull-Request Functions
+def getPRsFromAPI():
+    """ Gets all the pull-requests; Needs to use pagination because of 100 per-request api limit
+    """
+    PRs = []
+    pageNumber = 1
+    numberOfPRsReturned = 1
+    while numberOfPRsReturned != 0:
+        PRsResponse = requests.get(
+            "https://api.github.com/repos/greenelab/covid19-review/pulls?state=all&per_page=50&page=" +
```

Comment:
Do you want to ignore pull requests that were closed without being merged?  Open and merged pull requests would be most relevant to track.

Reply:
I think it depends on what the data in the table ends up being used for. It might be good to include all the relevant PR's including those that weren't merged so that, for instance, if someone proposed changes but the changes were rejected that information and discussion would still be linked to the paper. With the goal of trying to be as open as possible I think keeping all PR's may be the better bet.

Reply:
That logic makes sense to me